src/actions/get_all_events.js

```diff
@@ -0,0 +1,25 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+const getAllEvents = (hbs, response) => {
+  mockEvents.find({}, (error, events) => {
+    if (error) response.render('not_found')
+    else {
+      let upcomingEvents = [];
+      let pastEvents = [];
+      events.forEach((event) => {
+        if(event.eventDate > new Date()) {
```

Comment:
https://www.w3schools.com/js/tryit.asp?filename=tryjs_date_new
this is what new Date returns, this is how you wanna do it?